src/css/objects/_media.scss

```diff
@@ -112,21 +114,20 @@
 
     &--h3 {
       font-size: ms(3);
-      line-height: 1.222222222em;
+      line-height: 1.22222em;
     }
 
     &--h4 {
       font-family: $font-sans-base;
       font-size: ms(1);
```

Comment:
font size should be `ms(3)`, and a `$media-mobile` media query needs to be added to bring the font size down to `ms(2)` .